crates/ruma-client-api/src/dehydrated_device/put_dehydrated_device.rs

```diff
@@ -0,0 +1,90 @@
+//! `PUT /_matrix/client/*/dehydrated_device/`
+//!
+//! Uploads a dehydrated device to the homeserver.
+
+pub mod unstable {
+    //! `msc3814` ([MSC])
+    //!
+    //! [MSC]: https://github.com/uhoreg/matrix-doc/blob/shrivelled_sessions/proposals/3814-dehydrated-devices-with-ssss.md
```

Comment:
Same about linking to PR

Reply:
Fixed in dc5d4825971ba5abca39fc20f841e793b837af83.